plugins/discourse-local-dates/assets/javascripts/discourse/templates/components/discourse-local-dates-create-form.hbs

```diff
@@ -22,20 +39,30 @@
               {{i18n "discourse_local_dates.create.form.time_title"}}
             </label>
             <div class="controls">
-              {{input type="time" value=time class="time-input"}}
+              {{input input=(mut time) type="time" value=time class="time-input"}}
             </div>
           </div>
         </div>
 
-        <span class="to-indicator"><span>{{i18n "discourse_local_dates.create.form.to"}}</span></span>
+        <div class="to-indicator">
+          {{#if site.mobileView}}
```

Comment:
This looks like a good candidate for ember's inline-if

Reply:
indeed, I never use it 👍